src/app/portlets/content-types-create-edit/content-types-form/content-types-form.html

```diff
@@ -0,0 +1,98 @@
+                </div>
+                <div class="form__group">
+                  <label for="workflow" class="form__label">{{i18nMessages['Workflow']}}</label>
+                  <p-dropdown [options]="workflowOptions" id="content-type-form-workflow" name="workflow" formControlName="workflow"></p-dropdown>
+                </div>
+              </div>
+              <div class="ui-g-6">
+                <!--This will be only shown in edit-->
+                <!--<div class="form__group">
+                  <b>{{i18nMessages['Identifier']}}:</b>
+                  <a href="#">sdf3425-dsfd43-dfdsf-3342-vx</a>
+                  <i class="fa fa-question form__hint-icon" (click)="hint.show($event)"></i>
+                </div>
+                <div class="form__group">
+                  <div class="ui-g-6">
+                    <label for="publishDateField" class="form__label">{{i18nMessages['Publish-Date-Field']}}</label>
+                    <p-dropdown [options]="publishDateFieldOptions" id="content-type-form-publish-date-field" name="publishDateField" #publishDateField="ngModel" [(ngModel)]="contentTypeModel.publishDateField" placeholder="Select one"></p-dropdown>
+                  </div>
+                  <div class="ui-g-6">
+                    <label for="expireDateField" class="form__label">{{i18nMessages['Expire-Date-Field']}}</label>
+                    <p-dropdown [options]="expireDateFieldOptions" id="content-type-form-expire-date-field" name="expireDateField" #expireDateField="ngModel" [(ngModel)]="contentTypeModel.expireDateField" placeholder="Select one"></p-dropdown>
+                  </div>
+                </div>-->
+
+                <div class="form__group">
+                  <label for="detailPage" class="form__label">{{i18nMessages['Detail-Page']}}</label>
+                  <input pInputText type="text" id="content-type-form-detail-page" name="detailPage" formControlName="detailPage" />
+                </div>
+                <div class="form__group">
+                  <i class="fa fa-question form__hint-icon" (click)="hint.show($event)"></i>
+                  <label for="urlMapPattern" class="form__label">{{i18nMessages['URL-Pattern']}}</label>
+                  <input pInputText type="text" id="content-type-form-url-map-pattern" name="urlMapPattern" formControlName="urlMapPattern" />
+                </div>
+              </div>
+          </div>
+          <div class="form__buttons">
+            <button pButton type="button" flat id="content-type-form-cancel" label="{{i18nMessages['cancel']}}" (click)="goToListing()"></button>
+            <button pButton type="submit" id="content-type-form-submit" label="{{i18nMessages['save']}}"></button>
+          </div>
+        </div>
+      </form>
+      <div *ngIf="readyToAddFields">
+        <h1>Now you can add fields</h1>
```

Comment:
This is a placeholder text, will not exist after we add the drag and drop functionality for the fields